forms.py

```diff
@@ -57,11 +59,26 @@ class Meta:
 
 
 class DissertationRoleForm(ModelForm):
+
+    def clean(self):
+        data = self.cleaned_data
+        if dissertation_role.count_by_status_student_dissertation(data['status'],
+                                                                  data['adviser'],
+                                                                  data['dissertation']):
+            raise ValidationError('Error number of readers reached')
```

Comment:
On raise une ValidationError si on a déjà un DissertationRole avec le même statut + même adviser + même dissertation.
Ce n'est pas très cohérent avec le message d'erreur qui dit "nombre de lecteurs atteint"...

Reply:
Corrigé en "this reader has already been added"